candidate_models/base_models/__init__.py

```diff
@@ -335,8 +336,7 @@ class BaseModelPool(UniqueKeyDict):
     """
 
     def __init__(self):
-        super(BaseModelPool, self).__init__()
-        self._accessed_base_models = set()
+        super(BaseModelPool, self).__init__(reload=True)
```

Comment:
this seems like there should be some change to the `UniqueKeyDict` in `brain-score/submission`